build.ps1

```diff
@@ -331,7 +331,7 @@ function Append-FlagsDefine([hashtable]$Defines, [string]$Name, [string]$Value)
 }
 
 function Test-CMakeAtLeast([int]$Major, [int]$Minor, [int]$Patch = 0) {
-  $CMakeVersionString = @(cmake --version)[0]
+  $CMakeVersionString = @(& cmake.exe --version)[0]
```

Comment:
We also have `Invoke-Command` IIRC, but I suppose that we don't want to log this.

Reply:
Yeah, that was my thinking